play-json/shared/src/test/scala/play/api/libs/json/JsonTransSpec.scala

```diff
@@ -163,14 +163,14 @@ class JsonTransSpec extends WordSpec with MustMatchers {
     "report the correct path in the JsError" when {
       "the field to modify doesn't exist" in {
         js.transform(
-          (__ \ 'field42).json.update(__.read[JsString])
-        ).asEither.left.get.head mustEqual ((__ \ 'field42, Seq(JsonValidationError("error.path.missing"))))
+          (__ \ Symbol("field42")).json.update(__.read[JsString])
+        ).asEither.swap.getOrElse(???).head mustEqual ((__ \ Symbol("field42"), Seq(JsonValidationError("error.path.missing"))))
```

Comment:
Better to throw an error with a clear message instead of `???`. So, if the test fails, it will be easier to understand what is happening.

Reply:
Btw, it was throwing without a clear message before, with `.left.get`.